src/garage/__init__.py

```diff
@@ -1,4 +1,5 @@
 """Garage Base."""
 from garage._dtypes import TrajectoryBatch
+from garage.experiment.experiment import wrap_experiment
 
-__all__ = ['TrajectoryBatch']
+__all__ = ['wrap_experiment', 'TrajectoryBatch']
```

Comment:
hmm we should talk about if we want this in the `garage` package or the `garage.experiment` package -- it seems confusing to export it in both.

Reply:
I generally think we should put it in the `garage` package. I'm not completely certain what the function should be called though.